core/src/main/resources/swagger/api-docs.yaml

```
@@ -5742,12 +5742,12 @@ components:
         workspaceId:
           type: string
           description: A UUID associated with the workspace
-        googleProjectId:
+        googleProject:
           type: string
           description: Identifies the google project used by the workspace for compute and storage
```

At least in the description we can be specific about which identifier for the Google Project this is returning (as opposed to Google Project Number or Google Project Name)

```suggestion
          description: Google Project ID of the google project used by the workspace for compute and storage
```